assets/data-port/import/data/resolvers.js

```diff
@@ -0,0 +1,14 @@
+import { fetchFromAPI, setImportLog, errorFetchImportLog } from './actions';
+import { buildJobEndpointUrl } from '../helpers/url';
+
+export function* getLogsBySeverity( jobId ) {
```

Comment:
Why move this into a resolver? This seems like adding unnecessary complexity. This is not different from all the other fetch actions, let's keep them in the same place. Retrying in case of error also works by just dispatching the action again, not sure what `invalidateResolution` brings to the table. 
We also started to move toward less boilerplate/noise with the `composeFetchAction` helper - removing the repetitive action creators like start*, error*. (https://github.com/Automattic/sensei/pull/3305#discussion_r448493073)

Reply:
On second look I see this helps with fetching on demand, replacing the useEffect hook. So maybe as the best of both worlds, this resolver could just yield the original `fetchImportLog` action?

Reply:
Yep! With `@wordpress/data`, when it makes sense, I believe this format would be the more recommended because it already handles the fetching state, cache, and so on, for us. Unfortunately, it doesn't handle errors. Maybe at some point, we could study how to extend it to handle the errors too.

In the other cases that we're fetching, I think it doesn't make sense because the fetches are not necessarily tied to a selector, because we create a fetch for all the data and we consume when we need them. In this case that we are getting and consuming the data that we're fetching, I think it makes sense. =)

> So maybe as the best of both worlds, this resolver could just yield the original fetchImportLog action?

Good idea! Refactored here: https://github.com/Automattic/sensei/pull/3379/commits/25f3c60111116bb58201b2da8cc3ddf906c4899d
I just had to add a check in the `startAction`, because in this case we don't need that ;)